scripts/incentives-setup.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash -ex
+
+
+cleanup() {
+    kill $dualityd_pid >/dev/null 2>&1
+}
+trap cleanup ERR
+trap 'exit_code=$?; cleanup; exit $exit_code' EXIT
+
+rm -rf ~/.duality
+dualityd init local-node
+dualityd add-consumer-section
```

Comment:
I like this setup file as an example. but I don't want to overwrite the local state to set up a new chain. this is like having an alternate `setup.sh` file whereas really I would just like this to work with the regular setup, so I can have my regular faucet user and mnemonic and all those things.

I'd like to be able to run it in the https://github.com/duality-labs/dualityd-docker-services repo if I can. This would help:
- have an e2e test there
- let me run my standard local Docker Compose setup (including my price bot) with this as a modification

perhaps this epochs genesis state JSON could be a file I could optionally add into my setup.

I think an environment variable to pass in an arbitrary `GENESIS_STATE` to startup.sh would help. Or in the future devnet process: just having a "fast epoch" genesis JSON state I can load in.

Reply:
Open to all of these ideas. For now I think it's fine to leave this utility script here for future reference.